packages/flutter_tools/lib/src/base/build.dart

```diff
@@ -206,11 +206,11 @@ class AOTSnapshotter {
         .createSync(recursive: true);
     }
 
-    // Optimization arguments.
+    // Debugging information.
     genSnapshotArgs.addAll(<String>[
-      // Faster async/await
```

Comment:
these comments were outdated - previously the args added here were indeed related to optimization